packages/shared/src/types/abTests/epic.ts

```diff
@@ -39,6 +39,7 @@ export interface EpicVariant extends Variant {
     choiceCardAmounts?: ChoiceCardAmounts;
     defaultChoiceCardFrequency?: ContributionFrequency;
     bylineWithImage?: BylineWithImage;
+    articleCountCopy?: string;
```

Comment:
perhaps `articleCountCopy` could be added to the `SeparateArticleCount` type instead, since they're related

Reply:
I had the same thought, but decided to go with this rough implementation for now, as I wasn't sure how likely it was that we would add this functionality to the RRCP. I can make this change now though, for sure. We're still waiting on final sign-off for the epic copy on this test, so there's time.